pkg/web/templates/auth/login/login.html

```diff
@@ -0,0 +1,11 @@
+{{ template "auth.html" . }}
+
+{{ define "auth" }}
+<div class="auth-form p-3" hx-get="/v1/login" hx-swap="replace" hx-trigger="load">
```

Comment:
The change in Endeavor would be that this would have something like:

```
<div class="auth-form p-3" hx-get="{{ .PrepareLoginURL }}" hx-swap="replace" hx-trigger="load">
```